presidio-analyzer/install_dependencies.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ "$DEV_MODE" == "transformers" ]; then
+  poetry install -E server -E transformers --no-interaction
```

Comment:
Now that its a simpler condition, you can just pass the -E transformer as 'extra arguments' env var, and even drop the condition

Reply:
Other than that, looks great

Reply:
Way better :D
Deal!